images/stackrox-test.Dockerfile

```diff
@@ -49,6 +50,10 @@ RUN dnf update -y && \
     dnf clean all && \
     rm -rf /var/cache/dnf /var/cache/yum
 
+# Use updated auth plugin for GCP
+ENV USE_GKE_GCLOUD_AUTH_PLUGIN=True
+RUN gke-gcloud-auth-plugin --version
+
```

Comment:
@RTann shall I do this for scanner too? The changes look good on the stackrox side.

Reply:
yes, please, I noticed it for Scanner, too. Thanks!